lib/model/query/submissions.js

```diff
@@ -116,6 +116,8 @@ update.audit = (form, submission, data) => (log) => log('submission.update', for
 const clearDraftSubmissions = (formId) => ({ run }) =>
   run(sql`delete from submissions where "formId"=${formId} and draft=true`);
 
+const clearDraftSubmissionsForProject = (projectId) => ({ run }) =>
+  run(sql`DELETE FROM submissions USING forms WHERE submissions."formId" = forms.id AND forms."projectId" = ${projectId} AND submissions.draft=true`);
```

Comment:
It looks like deleted forms aren't encrypted (#945), yet this will delete the draft submissions of deleted forms. I don't think that's a big deal, and I don't think this function should change, but I thought I'd make a note of it.